e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/report-reconciliation/bank-utilisation-report-page.spec.js

```diff
@@ -38,14 +38,57 @@ context('Bank utilisation report page', () => {
     cy.task(NODE_TASKS.INSERT_TFM_FACILITIES_INTO_DB, matchingTfmFacilities);
   });
 
-  beforeEach(() => {
-    pages.landingPage.visit();
-    cy.login(USERS.PDC_RECONCILE);
+  describe(`when navigating to the page as ${USERS.PDC_RECONCILE}`, () => {
+    beforeEach(() => {
+      pages.landingPage.visit();
+      cy.login(USERS.PDC_RECONCILE);
 
-    cy.visit(`utilisation-reports/${reportId}`);
+      cy.visit(`utilisation-reports/${reportId}`);
+    });
+
+    it('should display the page heading and date', () => {
+      cy.assertText(utilisationReportPage.heading(), bankName);
+
+      cy.assertText(utilisationReportPage.reportPeriodHeading(), reportPeriodString);
+    });
+
+    it('should display the correct tabs', () => {
+      cy.assertText(utilisationReportPage.premiumPaymentsTabLink(), 'Premium payments');
+      cy.assertText(utilisationReportPage.keyingSheetTabLink(), 'Keying sheet');
+      cy.assertText(utilisationReportPage.paymentDetailsTabLink(), 'Payment details');
+      cy.assertText(utilisationReportPage.utilisationTabLink(), 'Utilisation');
+    });
+
+    it('should display the correct text', () => {
+      cy.assertText(
+        utilisationReportPage.premiumPaymentsTab.howToAddPaymentsText(),
+        `Received payments are entered against reported fees through selection and then selection of the 'Add a payment' button.`,
+      );
+
+      cy.assertText(
+        utilisationReportPage.premiumPaymentsTab.howToGenerateKeyingDataText(),
+        `When payments show as matched, the adjustment data for keying into ACBS will be automatically generated when the 'Generate keying data' button is selected.`,
+      );
+
+      cy.assertText(
+        utilisationReportPage.premiumPaymentsTab.createRecordCorrectionRequestText(),
+        `If there is an error with the fee record, select the record to create a record correction request to send a query to the bank.`,
+      );
```

Comment:
We also will need tests in the feature flag off folder that this does not show when the feature flag is turned off